packages/dev/docs/pages/react-aria/collections.mdx

```diff
@@ -0,0 +1,315 @@
+the items that changed as needed.
+
+#### useTreeData
+
+The <TypeLink links={dataDocs.links} type={dataDocs.exports.useTreeData} /> hook can be used to
+manage data and state for a tree of items. This is similar to `useListData`, but with support for
+heirarchical data. Like `useListData`, `useTreeData` will also handle automatically
+removing items from the selection when they are removed from the list.
+See the [useTreeData](../react-stately/useTreeData.html) docs for more details.
+
+```tsx
+import {useTreeData} from '@react-stately/data';
+
+let tree = useTreeData({
+  initialItems: [
+    {
+      name: 'People',
+      items: [
+        {name: 'David'},
+        {name: 'Sam'},
+        {name: 'Jane'}
+      ]
+    },
+    {
+      name: 'Animals',
+      items: [
+        {name: 'Aardvark'},
+        {name: 'Kangaroo'},
+        {name: 'Snake'}
+      ]
+    }
+  ],
+  getKey: item => item.name,
+  getChildren: item => item.items
+});
+
+function addPerson(name) {
+  tree.append('People', {name});
+}
+
+<ListBox items={tree.items}>
+  {node =>
+    <Section id={section.name} items={node.children}>
+      <Header>{section.name}</Header>
+      <Collection items={section.children}>
+        {item => <ListBoxItem id={item.name}>{item.name}</ListBoxItem>}
+      </Collection>
+    </Section>
+  }
+</ListBox>
+```
+
+Note that `useTreeData` is a utility hook, not a requirement. You can use any state management library to manage collection items.
+
+
+## Asynchronous loading
+
+The <TypeLink links={dataDocs.links} type={dataDocs.exports.useAsyncList} /> hook can be
+used to manage async data loading from an API. Pass a `load` function to `useAsyncList`, which returns the items
+to render. You can use whatever data fetching library you want, or the built-in
+[fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) API. See the [useAsyncList](../react-stately/useAsyncList.html) docs for more details.
+
+This example fetches a list of Pokemon from an API and displays them in a Select.
+
+```tsx
+import {useAsyncList} from '@react-stately/data';
+
+let list = useAsyncList({
+  async load({signal}) {
+    let res = await fetch('https://pokeapi.co/api/v2/pokemon', {signal});
+    let json = await res.json();
+    return {items: json.results};
+  }
+});
+
+<Select>
+  <Label>Pick a Pokemon</Label>
+  <Button>
+    <SelectValue />
+  </Button>
+  <Popover>
+    <ListBox items={list.items}>
+      {item => <ListBoxItem id={item.name}>{item.name}</ListBoxItem>}
+    </ListBox>
+  </Popover>
+</Select>
+```
+
+Note that `useAsyncList` is a convenience hook, not a requirement. You can use any state management or data loading
+library to manage collection items.
+
+## Hooks
+
+If you're using React Aria and React Stately hooks rather than components, the collection API is slightly different.
+See the [React Stately collections](../react-stately/collections.html) documentation for more details.
```

Comment:
mega nitpick: I feel like this disclaimer should be higher up in the page, it isn't immediately obvious that the "hooks" section in the table of contents is referring to our aria/stately hooks nor that this page is RAC specific.

Reply:
We are emphasizing the components on all of the pages now, and a few have a hooks section at the bottom. If this turns out to be confusing for people, we can always update it.